charts/zitadel/templates/pdb.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.pdb.enabled }}
+{{- if .Values.pdb.apiVersion }}
+apiVersion: {{ .Values.pdb.apiVersion }}
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
```

Comment:
Can we handle this like we do in the ingress template?

Reply:
That's fancy! Unsure if this is the correct way to implement, but I tried nonetheless